deployment/k8s/charts/components/bootstrapHelm/templates/bootstrap-job.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if not .Values.skipImageBuild }}
```

Comment:
Can you rename this chart to something like buildImages please?  helm install bootstrapHelm is pretty loopy :)

Reply:
Done